src/tests/unit/tests/debug-tools/debug-tools-message-distributor.test.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { BrowserAdapter } from 'common/browser-adapters/browser-adapter';
+import { StoreUpdateMessageHub } from 'common/store-update-message-hub';
+import { StoreUpdateMessage } from 'common/types/store-update-message';
+import { TelemetryListener } from 'debug-tools/controllers/telemetry-listener';
+import { DebugToolsMessageDistributor } from 'debug-tools/debug-tools-message-distributor';
+import { IMock, It, Mock } from 'typemoq';
+
+describe(DebugToolsMessageDistributor, () => {
+    let registeredListener: (message: any) => void;
+    let browserAdapterMock: IMock<BrowserAdapter>;
+    let telemetryListenerMock: IMock<TelemetryListener>;
+    let storeUpdateHubMock: IMock<StoreUpdateMessageHub>;
+
+    let testSubject: DebugToolsMessageDistributor;
+
+    beforeEach(() => {
+        browserAdapterMock = Mock.ofType<BrowserAdapter>();
+        telemetryListenerMock = Mock.ofType<TelemetryListener>();
+        storeUpdateHubMock = Mock.ofType<StoreUpdateMessageHub>();
+
+        browserAdapterMock
+            .setup(b => b.addListenerOnMessage(It.isAny()))
+            .returns(listener => (registeredListener = listener));
```

Comment:
nit: doesn't matter functionally since we check if the registered listener is defined later, but it might be more clear if you verify that this is called exactly once after init too.